<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bdAnimationToolWin</class>
 <widget class="QMainWindow" name="bdAnimationToolWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Settings</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Wip Animation Folder:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="wipFolderEdit"/>
              </item>
              <item>
               <widget class="QPushButton" name="wipAnimBtn">
                <property name="text">
                 <string>Browse</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="openFolderBtn">
                <property name="text">
                 <string>Open Folder</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Playblast camera</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="playblastCamEdit"/>
              </item>
              <item>
               <widget class="QPushButton" name="playblastCamBtn">
                <property name="text">
                 <string>Pick</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Playblast format</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="animFormatComboBox">
                <item>
                 <property name="text">
                  <string>mov</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>avi</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Folders</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLineEdit" name="charNameEdit">
                <property name="text">
                 <string>Enter character name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="animTypeCombo">
                <item>
                 <property name="text">
                  <string>entry</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>idle</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>action</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>exit</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="newAnimBtn">
                <property name="text">
                 <string>Create New Animation</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="1" column="0">
               <widget class="QTableWidget" name="wipAnimTable">
                <column>
                 <property name="text">
                  <string>Animation File</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Version</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Has playblast ?</string>
                 </property>
                </column>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Preview</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Wip</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QTableWidget" name="previewAnimTable">
                <column>
                 <property name="text">
                  <string>Animation File</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Version</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Has Playblast ?</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="lastVersionCheckBox">
              <property name="text">
               <string>Wip Last Versions Only</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>File</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QPushButton" name="openFileBtn">
              <property name="text">
               <string>Open Selected</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="saveFileBtn">
              <property name="text">
               <string>Save New Version</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="copyToPreviewBtn">
              <property name="text">
               <string>Copy Wip To Preview</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>Playblast</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QPushButton" name="playblastBtn">
              <property name="text">
               <string>Playblast current scene</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="copyPlayblastBtn">
              <property name="text">
               <string>Copy playblast to preview</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
